packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -19,7 +19,8 @@
 
 {# Attributes #}
 {%- set _dataImagePreviewAttr = _generateImagePreview ? 'data-spirit-imagePreview="true"' : null -%}
-{%- set _imageObjectFitAttr = _imageObjectFit ? 'data-spirit-image-object-fit=' ~ _imageObjectFit : null -%}
+{%- set _imageObjectFitCSSAttr = '--file-uploader-attachment-image-object-fit:' -%}
+{%- set _imageObjectFitAttr = _imageObjectFit ? 'style=' ~ _imageObjectFitCSSAttr ~ _imageObjectFit : null -%}
```

Comment:
If Prettier allows, the line with the image (out of scope of this PR) could be rather expanded to multiple lines:

```twig
<img
  src="{{ _imagePreview }}"
  {{ _imageObjectFitAttr | raw }}
  width="54"
  height="54"
  alt="{{- _fileName -}}"
/>
```

Reply:
done